README.md

```diff
@@ -101,6 +101,8 @@ assert batch['text'][0][0:8] == ['Wall', 'St.', 'Bears', 'Claw', 'Back', 'Into',
 python setup.py install
 ```
 
+In you'd like to include the S3 IO datapipes and aws-sdk-cpp, you may also follow [the instructions here](https://github.com/pytorch/data/blob/1d1cdbefe38041e067cda835ced9b5a5c59b3e5b/torchdata/datapipes/iter/load/README.md#build-from-source)
```

Comment:
Could you please use `https://github.com/pytorch/data/blob/main/torchdata/datapipes/iter/load/README.md` to always link it to the main branch?

Reply:
Updated.